src/main/steps/task-list/utils.ts

```diff
@@ -126,3 +126,32 @@ export const getAdoptionCertificateDetailsStatus = (userCase: CaseWithId): Secti
     ? SectionStatus.NOT_STARTED
     : SectionStatus.IN_PROGRESS;
 };
+
+const addressComplete = (userCase: CaseWithId, fieldPrefix: FieldPrefix) => {
+  const address1 = userCase[`${fieldPrefix}Address1`];
+  const addressTown = userCase[`${fieldPrefix}AddressTown`];
+  const addressPostcode = userCase[`${fieldPrefix}AddressPostcode`];
+  const addressCountry = userCase[`${fieldPrefix}AddressCountry`];
+
+  return (address1 && addressTown && addressPostcode) || (address1 && addressCountry);
+};
+
+export const getOtherParentStatus = (userCase: CaseWithId): SectionStatus => {
+  const exists = userCase.otherParentExists;
+  const names = userCase.otherParentFirstNames && userCase.otherParentLastNames;
```

Comment:
I think this logic needs some correction
we need to check names only when exists == YesOrNo.YES